tests/bootstrap.php

```diff
@@ -1,11 +1,5 @@
 <?php
 
-// backward compatibility
-if (!class_exists('\PHPUnit\Framework\TestCase') &&
-    class_exists('\PHPUnit_Framework_TestCase')) {
-    class_alias('\PHPUnit_Framework_TestCase', '\PHPUnit\Framework\TestCase');
-}
-
```

Comment:
Since the PHPUnit uses the `6.4` version at least, the default PHPUnit namespace is `PHPUnit\Framework\TestCase`.

The `class_alias` and `class` checking are not necessary.